api/migrations/20210309170828-redactPublicNames.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+let dbm;
+let type;
+let seed;
+
+const RecordTypeEnum = require('../src/utils/constants/misc');
+
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function(db) {
+
+  const mClient = await db.connection.connect(db.connectionString, {
+    native_parser: true
+  });
+
+  try {
+
+    console.log('**** Started redacting individuals names ****');
+
+    const nrpti = await mClient.collection('nrpti');
+
+    const recordSchemas = [
+      ...RecordTypeEnum.MASTER_SCHEMA_NAMES,
+      ...RecordTypeEnum.LNG_SCHEMA_NAMES,
+      ...RecordTypeEnum.NRCED_SCHEMA_NAMES,
+      ...RecordTypeEnum.BCMI_SCHEMA_NAMES
+    ];
+
+    const noPublishIssuingAgencies = [
```

Comment:
Maybe use the values from the `AUTHORIZED_PUBLISH_AGENCIES` constant instead of creating the inverse of it?  This avoids some issues such as inconsistent issuing agency names between API and UI (EMLI is stored as `EMLI` in the DB).  

https://github.com/bcgov/NRPTI/blob/master/api/src/utils/constants/misc.js#L164-L170

Reply:
Good Idea, thanks!